### ISSUE TITLE
Bogus achievements

### ISSUE THREAD

**Issue by SabreCat** [HabitRPG/habitica#1889]
UID 7f14ed62-5408-4e1b-be83-ada62d504931

I have a mishmash of different tiers of gear, and 70 pets. But my profile is showing the Beastmaster and Ultimate Gear achievements:

![screenclip](https://f.cloud.github.com/assets/4501321/1605915/3e1c322e-5402-11e3-9524-f1333146c857.png)

This appears in the JS console:

```
[23:42:49.226] "Error: [$compile:ctreq] Controller 'bindonce', required by directive 'boClass', can't be found!
http://errors.angularjs.org/1.2.2/$compile/ctreq?p0=bindonce&p1=boClass
d/<@https://beta.habitrpg.com/app-79f3cdbb.js:4
ib@https://beta.habitrpg.com/app-79f3cdbb.js:5
jb@https://beta.habitrpg.com/app-79f3cdbb.js:5
p@https://beta.habitrpg.com/app-79f3cdbb.js:5
p@https://beta.habitrpg.com/app-79f3cdbb.js:5
g@https://beta.habitrpg.com/app-79f3cdbb.js:5
c@https://beta.habitrpg.com/app-79f3cdbb.js:5
J@https://beta.habitrpg.com/app-79f3cdbb.js:5
j@https://beta.habitrpg.com/app-79f3cdbb.js:7
g.prototype.$digest@https://beta.habitrpg.com/app-79f3cdbb.js:6
g.prototype.$apply@https://beta.habitrpg.com/app-79f3cdbb.js:6
y/<.link/<@https://beta.habitrpg.com/app-79f3cdbb.js:8
u.event.dispatch@https://beta.habitrpg.com/app-79f3cdbb.js:2
u.event.add/s.handle@https://beta.habitrpg.com/app-79f3cdbb.js:2
" "<div bo-class="{muted: !profile.achievements.beastMaster}">"
```

**Comment by SabreCat** [comment]
Looks related to @snicker's #1885 ?

**Comment by hattieshoe** [comment]
I just noticed I have Beast Master too despite only having 51 pets.

**Comment by hattieshoe** [comment]
Though, strangely, when I click on my name by a comment and the box comes up with my information, Beast Master does not show up there. (It's there but grayed out as it always has been.) But on my Stats and Achievements page it shows the Beast Master achievement. SabreCat, is that the way yours is happening?

**Comment by SabreCat** [comment]
@hattieshoe: No, I first noticed it when clicking on my own name in Tavern chat--it's wrong both places, here.

**Comment by snicker** [comment]
This is related to #1885 yes. i've been rolling back some of the changes here and there #1886, #1891, etc... sorry bout that folks

**Comment by snicker** [comment]
this one should be fixed by #1886, needs to be made live on beta
